.travis.yml

```diff
@@ -0,0 +1,14 @@
+branches:
+  except:
+    - master
+language: ruby
+cache: bundler
+sudo: false
+rvm:
```

Comment:
I had missed your reply [to my previous comment](https://github.com/cerner/cerner_splunk_ingredient/pull/1#discussion_r65640587) but we should indeed be running tests under ruby 2.1.x lest we have some syntax or quirk that doesn't jive with the version of ruby that ships with Chef. 

Guard seems like a utility that you'd use only with local development... should it be in the bundle? Or should we have a way to exclude it for Travis builds? Also there's an interesting note on https://github.com/guard/guard saying that with the right version of bundler, it should install the appropriate version for your ruby... maybe that's something to look into as well?